tests/scancode/data/help/help.txt

```diff
@@ -22,19 +22,24 @@ Options:
     --license-score INTEGER      Do not return license matches with a score lower
                                  than this score. A number between 0 and 100.
                                  [default: 0]
+
```

Comment:
@pombredanne : This has been done by `regen`. but then also it failing on ci but not on my mac.